pkgs/development/libraries/nss/generic.nix

```diff
@@ -45,13 +40,6 @@ stdenv.mkDerivation rec {
   propagatedBuildInputs = [ nspr ];
 
   prePatch = ''
```

Comment:
This should really be moved all to postPatch otherwise some patches might not apply.